.github/workflows/tag.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         id: go
 
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
```

Comment:
This breaks deployment. By default goreleaser expects the full history but from v1 to v2 they started only downloading to current commit.